projects/storefrontlib/layout/main/storefront.component.ts

```diff
@@ -92,6 +92,7 @@ export class StorefrontComponent implements OnInit, OnDestroy {
     useFeatureStyles('a11yKeyboardFocusInSearchBox');
     useFeatureStyles('a11yNgSelectLayering');
     useFeatureStyles('topProgressBarUseTransformAnimation');
+    useFeatureStyles('reserveSpaceForImagesOnPdpAndPlp');
```

Comment:
Can we add this line instead only to the 2 constructors of 2 components that we're affecting?
- ProductImagesComponent
- ProductListComponent